config/config.go

```diff
@@ -30,11 +30,26 @@ type Config struct {
 		CaCertFile string `envconfig:"CA_CERT_FILE" default:"/tmp/certs/ca-cert.pem"`         // CA certificate file
 		Insecure   bool   `envconfig:"CLIENT_INSECURE" default:"true"`                        // don't check server certificate
 	}
+
+	DelegateCapacity struct {
+		ManagerEndpoint string `envconfig:"MANAGER_ENDPOINT" default:"http://localhost:3457"`
```

Comment:
IMO we can remove the default values since anyway register delegate capacity is a hit-or-miss. And also, it's not needed in a local setup.